example/config/config.yaml

```diff
@@ -28,29 +28,35 @@ index_service:
         "Validator",
         "ValidatorStats",
         "NFT",
-      #      "CryptoComNFT",
+        #      "CryptoComNFT",
         "IBCChannel",
-      #      "IBCChannelTxMsgTrace",
+        #      "IBCChannelTxMsgTrace",
         "IBCChannelMessage",
         "BridgePendingActivity",
         "Example",
-    ]
+        "ValidatorDelegation",
+      ]
     extra_configs:
       BridgePendingActivity:
         this_chain_id: "testnet-croeseid-4"
         this_chain_name: "Crypto.org-Chain"
         counterparty_chain_name: "Cronos"
         channel_id: "channel-131"
         starting_height: 899374
+      ValidatorDelegation:
+        unbonding_time: "2419200000000000ns"
+        slash_fraction_double_sign: "0.050000000000000000"
```

Comment:
May i know that are those slash fractions can be change by proposal?

Reply:
https://crypto.org/explorer/proposals

You can check those proposals there. So there are different type of proposals. And there are module params that could be configured through `gov` module's proposal.